tf/scripts/frontend_user_data.sh

```
@@ -1,20 +1,82 @@
+  -p 3000:3000 \
+  -e NODE_ENV=production \
+  -e REACT_APP_BACKEND_URL=http://${backend_private_ip}:8080 \
+  -e DD_SERVICE=lti-frontend \
+  -e DD_ENV=${environment} \
+  -e DD_VERSION=${app_version} \
+  lti-frontend
+
+# ==========================================
+# Datadog Agent Installation
+# ==========================================
+DD_API_KEY="${datadog_api_key}" \
```

Shellcheck (0.11.0)

[warning] datadog_api_key is referenced but not assigned.

(SC2154)